tests/main/cgroup-freezer/task.yaml

```diff
@@ -0,0 +1,24 @@
+summary: Each snap process is moved to appropriate freezer cgroup
+details: |
+    This test creates a snap process that suspends itself and ensures that it
+    placed into the appropriate hierarchy under the freezer cgroup.
+prepare: |
+    . $TESTSLIB/snaps.sh
+    install_local test-snapd-sh
+execute: |
+    test-snapd-sh -c 'sleep 1000' &
+    pid=$!
```

Comment:
Maybe it would be worth starting two instances of the command in the test? That would give coverage for adding a process to an existing cgroup.

Reply:
Sure, I'll do that.

Reply:
Done, it took me forever to get right (due to wait and lack of exec sleep) but I figured it out and now it works :)